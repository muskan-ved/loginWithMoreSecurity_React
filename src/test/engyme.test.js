import { render, fireEvent } from '@testing-library/react';
import { shallow } from 'enzyme';
import TestPage from '../Jest test/testPage';
 
describe('The Counter component', () => {
  describe('if the button is not clicked', () => {
    it('should display 0', () => {
      const counter = render(<TestPage />);
      const paragraph = counter.getByText('This is a test page', {
        selector: 'p',
      });
 
      expect(paragraph).toBeDefined();
    });
  });

  
  
  it('object check', () => {
      const wrapper = shallow(<TestPage />);
      console.log(wrapper.debug());
      expect(wrapper).toMatchSnapshot();
      expect(wrapper.exists('.row')).toEqual(true);
    })
});