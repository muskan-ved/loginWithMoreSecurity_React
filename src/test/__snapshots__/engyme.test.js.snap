// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`The Counter component object check 1`] = `ShallowWrapper {}`;
